### PR TITLE
Initial commit for parsnp recipe

### PR DESCRIPTION
**Pull request by thomcuddihy**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Code review requested, please @bioconda/core

### CODE REVIEW

File: recipes/parsnp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+  run:
+    - zlib
+
+about:
+  home: https://github.com/marbl/parsnp
+  license: "https://raw.githubusercontent.com/marbl/parsnp/master/LICENSE"
```
Comment (rvalieris) [inline recipes/parsnp/meta.yaml]: please, change this line to license_file:
and add a line license: custom

**Comment by thomcuddihy** [comment]
@rvalieris using `license_file` triggers the build system to try to copy that field to the output directory. Unfortunately the LICENSE file is not included in the archive (for `license_file: LICENSE`), and build system is not able to handle URLs. Have reverted to specifying the license URL in the `license` field`, along with a `custom` identifier, similar to the UCSC recipes.

**Comment by rvalieris** [comment]
oh thats unfortunate, but alright. please do that change on the other recipe as well, harvesttools.

another thing, you are not compiling anything here, so the compiler is not required ? can you remove it ?

**Comment by thomcuddihy** [comment]
Completed @rvalieris

**Comment by rvalieris** [comment]
thanks, sorry for the delay.

**Pull request merged**